scripts/mkcloud

```diff
@@ -967,6 +974,7 @@ Steps:
     setupnodes:     create the nodes and let crowbar discover them
     instnodes:      allocate and install compute nodes
     proposal:       create and apply proposals for default setup
+    setup_aliases:  set aliases for the nodes (usually needed for batch step)
```

Comment:
I prefer `batch` - less typing, and still everyone knows what it does.  Plus everything in `mkcloud` is related to Crowbar anyway, so the `crowbar_` prefix seems redundant to me.  Why do you want the longer version?